<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patchmatch</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="ImageView" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMove"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionReshuffleRectangle"/>
   <addaction name="actionReshuffleFreeHand"/>
   <addaction name="separator"/>
   <addaction name="actionReplaceRectangle"/>
   <addaction name="actionReplaceFreeHand"/>
   <addaction name="separator"/>
   <addaction name="actionRetarget"/>
   <addaction name="actionProcess"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/tool-move.png</normaloff>:/icons/icons/tool-move.png</iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/tool-delete.png</normaloff>:/icons/icons/tool-delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionReshuffleRectangle">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/tool-rect-select.png</normaloff>:/icons/icons/tool-rect-select.png</iconset>
   </property>
   <property name="text">
    <string>Reshuffle Rectangle</string>
   </property>
  </action>
  <action name="actionReshuffleFreeHand">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/tool-free-select.png</normaloff>:/icons/icons/tool-free-select.png</iconset>
   </property>
   <property name="text">
    <string>Reshuffle Free Hand</string>
   </property>
  </action>
  <action name="actionReplaceRectangle">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/tool-rect-select2.png</normaloff>:/icons/icons/tool-rect-select2.png</iconset>
   </property>
   <property name="text">
    <string>Replace Rectangle</string>
   </property>
  </action>
  <action name="actionReplaceFreeHand">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/tool-free-select2.png</normaloff>:/icons/icons/tool-free-select2.png</iconset>
   </property>
   <property name="text">
    <string>Replace Free Hand</string>
   </property>
  </action>
  <action name="actionRetarget">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/tool-retarget.png</normaloff>:/icons/icons/tool-retarget.png</iconset>
   </property>
   <property name="text">
    <string>Retarget</string>
   </property>
  </action>
  <action name="actionProcess">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/process.png</normaloff>:/icons/icons/process.png</iconset>
   </property>
   <property name="text">
    <string>Process</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QWidget</extends>
   <header>imageview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
